### ISSUE TITLE
Typos in HydroData:::col_lc

### ISSUE THREAD

**Issue by jsta** [mikejohnson51/HydroData#23]
Would a PR fixing mis-capitalization in `woodywetlands` and an incorrect description field for code 82 be welcome?

**Comment by mikejohnson51** [comment]
Hi Joseph,

Help/contributions are always welcome! Sorry I have been away from my main computer for a bit. The typos are definitly a 'must' fix... I also saw your color table suggestion. I pulled the colors a couple years ago and don't remember the exact source but know they came from a USGS page. They have generally looked right to me... I am attaching a file I have been using recently...

Have you found the new ones to be better? Lets sort that out and then I'll merge these.

Thanks!

![land_use_nwm](https://user-images.githubusercontent.com/30052272/47669069-4628e600-db67-11e8-8098-62794d4eb541.gif)
